specification/hugr.md

```diff
@@ -153,7 +185,7 @@ A **sibling graph** is a subgraph of the HUGR containing all nodes with
 a particular parent, plus the Order, Value and ControlFlow edges between
 them.
 
-#### Value
+##### Value
```

Comment:
heading level 5 results in text that is even smaller than the body on github rendering, I think either move it back up a level or just use Bold for these Kind names

Reply:
OK. (Using excessive levels of heading is a bad habit of mine.)